zone/dialogue_window.cpp

```diff
@@ -98,10 +99,10 @@ void DialogueWindow::Render(Client *c, std::string markdown)
 		}
 	}
 
-	uint32 popup_id       = POPUPID_DIAWIND;
-	uint32 negative_id    = 0;
-	char   *button_name_0 = nullptr;
-	char   *button_name_1 = nullptr;
+	uint32 popup_id = POPUPID_DIAWIND_ONE;
+	uint32 negative_id = POPUPID_DIAWIND_TWO;
+	const char *button_one_name = nullptr;
```

Comment:
Any reason we didn't just make these `std::string`'s and then translate them to `c_str` primitives later?